### PR TITLE
chore(release): pull release/v1.54.0 into main

### PR DESCRIPTION
**Pull request by devops-github-rudderstack**
:crown: *An automated PR*

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3017/test-report.html

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,41 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.54.0](https://github.com/rudderlabs/rudder-transformer/compare/v1.53.0...v1.54.0) (2024-01-22)
+
```
Comment (krishna2020) [inline CHANGELOG.md:6]: Deduplicate the changelog - due to missing commits in `main` branch, 1.53.0 version changes got duplicated

Comment (yashasvibajpai) [reply CHANGELOG.md:6]: updated

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3017/test-report.html

**Review by koladilip — APPROVED** [review]

**Review by krishna2020 — APPROVED** [review]

**Pull request merged**